pages/Configuring/Dispatchers.md

```diff
@@ -85,6 +85,14 @@ layout pages (See the sidebar).
 
 {{< callout type=warning >}}
 
+uwsm users should avoid using `exit` dispatcher, as exiting Hyprland this way causes various systemd-related problems after restarting the compositor, such xdg-desktop-portal-hyprland and other services do not restart. Use `loginctl terminate-user ""` instead.
```

Comment:
It shouldn't cause those kind of problems, just a bunch of errors in apps due to sudden compositor disappearance. Could you file a bug to uwsm with logs?

Reply:
I can check it, later, but since compositor-site exit isn't a desirable way to exit the session with uwsm, I think we should leave the warning

Reply:
Warning/note on how to properly exit is valid.
Scope of consequences needs checking.

Reply:
What logs do you need?

Reply:
`journalctl --user` when exiting via dispatcher and then starting again.

Reply:
https://github.com/Vladimir-csp/uwsm/issues/51